spec/rails_helper.rb

```diff
@@ -14,8 +14,8 @@
   c.hook_into :webmock
   c.ignore_localhost = true
   c.configure_rspec_metadata!
-  c.filter_sensitive_data("<GITHUB_CLIENT_ID>") { ENV['ST_GITHUB_CLIENT_ID'] }
-  c.filter_sensitive_data("<GITHUB_CLIENT_SECRET>") { ENV['ST_GITHUB_CLIENT_SECRET'] }
+  c.filter_sensitive_data("<GITHUB_CLIENT_ID>") { ENV['GITHUB_CLIENT_ID'] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.